### DISCUSSION TITLE
Add Ability to Override Ignore SHAs

### DISCUSSION THREAD

**Discussion by JaydenMaalouf** [GitTools/GitVersion#4098] [Ideas]
Hello,
I think it would be useful to be able to override `ignore__shas` as an argument in the commandline
The use case behind this is I am using this in a monorepo and make use of `tag-prefix` already to tag commits separately based on the affected folders
However, I would like to ignore commits where those folders were never changed (but other folders were changed inbetween the last version number) - which I'm able to capture with `git rev-list --ancestry-path $LAST_SHA..HEAD -- . ":!$PATH/"`

Thanks heaps!

**Comment by HHobeck** [comment]
@arturcic Is it correct that with the refactoring initiative of GitVersion version 7.0.0 all configuration properties can be overwritten via command line arguments?

However I think this issue goes in the same direction:
- #2436

**Reply by asbjornu** [reply]
The plan for the new CLI is that every configuration property should be overridable by a command line argument, yes.
